cirq/contrib/acquaintance/executor.py

```diff
@@ -93,6 +111,19 @@ def optimization_at(self,
                          'are instances of AcquaintanceOpportunityGate or '
                          'PermutationGate.')
 
+
+class AcquaintanceOperation(ops.GateOperation):
+    """Represents an a realized acquaintance opportunity.
```

Comment:
This description is not very clear. What is a "realized acquaintance opportunity"?

Is this class an operation that occurs as two qubits pass each other during a permutation, including details both of the physical qubits being acted upon and the logical qubits/modes those physical qubits are temporarily storing?

Reply:
Yea, that's right. I updated the docstring.